package.json

```diff
@@ -9,6 +9,7 @@
     "build": "tsc",
     "build:live": "nodemon --exec ./node_modules/.bin/ts-node -- ./src/index.ts",
     "build-and-publish": "tsc && npm publish --access public",
+    "build-and-publish-local": "tsc && npm publish --registry http://localhost:4873/",
```

Comment:
prerequisite is a running local registry, right? 
I would add this script in a dedicated pull request with the required npm package